pkg/image/importer/importer.go

```diff
@@ -397,6 +418,7 @@ func importRepositoryFromDocker(ctx gocontext.Context, retriever RepositoryRetri
 			importDigest.Err = err
 			continue
 		}
+		importDigest.Err = calculateImageSize(ctx, repo, importDigest.Image)
```

Comment:
It doesn't matter that it's not documented - it's what is deployed in
production in many places still.

On Wed, Jun 1, 2016 at 1:52 PM, Alexey Gladkov <notifications@github.com>
wrote:

> In pkg/image/importer/importer.go
> <https://github.com/openshift/origin/pull/9115#discussion_r65409920>:
>
> > @@ -397,6 +418,7 @@ func importRepositoryFromDocker(ctx gocontext.Context, retriever RepositoryRetri
> >  			importDigest.Err = err
> >  			continue
> >  		}
> > +		importDigest.Err = calculateImageSize(ctx, repo, importDigest.Image)
>
> Schema v1 supported by hub.docker.com. Before that we used is not
> documented structure to calculate the size.
>
>
> https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-1.md#manifest-field-descriptions
>
> v1Compatibility string
>
> V1Compatibility is the raw V1 compatibility information. This will contain the JSON
> object describing the V1 of this image.
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9115/files/492baa03f23cf45ca229c3501c29ca4bccf546f3#r65409920>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p0YnP3kHQqbqW47WCSl_-TNMETGtks5qHcbcgaJpZM4IrzC2>
> .
>